contrib/libmd/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "libmd"
+pkgver = "1.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = []
```

Comment:
```suggestion
# why
configure_gen = []
```

Reply:
to get things like --prefix=/usr and so on, by default gnu_configure runs `autoreconf -if` which in this repo doesn't make sense.

Reply:
i'm aware, that's what the comment is for

Reply:
though now it seems like this is actually autotools, is this a new source, i'm confused haha